mitodl/bootcamp-ecommerce
# Handle webhooks from FluidReview

#### What are the relevant tickets?
Part of #136 

#### What's this PR do?
Checks the authorization token for fluidreview webhooks, then stores the content in the database

#### How should this be manually tested?
 - Set `FLUIDREVIEW_WEBHOOK_AUTH_TOKEN` to some value in `.env`
 - Run this command: `curl http://bootcamp.mit.local:8099/api/v0/fluidreview_webhook/ -H "Authorization: Basic wrong_token" -X POST --data '{"a": "b"}'`, substituting your token and URL. You should see an error message saying that you don't have permission to do this.
 - Next, run the same command, but replacing your token in the command line. In the Django shell check that there is only one `WebhookRequest` object in the database and verify that its contents match `{"a": "b"}`
